### PR TITLE
Fix news

### PR DESCRIPTION
**Pull request by VolodymyrTrach**

### CODE REVIEW

File: .env.exaple [hunk 1]
```diff
@@ -1 +1,2 @@
 SKIP_PREFLIGHT_CHECK=true
+BASE_URL
```
Comment (Alan18081) [inline .env.exaple:2]: Please, add example value that points to localhost

File: src/pages/news/news-item/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import NewsPageItem from './news-item';
+
```
Comment (Alan18081) [inline src/pages/news/news-item/index.js:2]: Please, rename component here too

File: src/pages/news/news-page/news.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 import { LANGUAGE } from '../../../configs';
 
-const NewsPage = ({ watchNewsLoad, list }) => {
+const NewsPage = ({ getNews, list }) => {
   useEffect(() => {
-    watchNewsLoad();
+    getNews();
     window.scrollTo(0, 0);
-  }, [watchNewsLoad]);
+  }, [getNews]);
```
Comment (Alan18081) [inline src/pages/news/news-page/news.js:12]: Why do we pass getNews that is a function? Is it going to change?

File: src/redux/news/news.reducer.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ const list = (state = initialState, action) => {
         ...state,
         list: action.payload
       };
-    case NEWS_LOADED:
+    case GET_NEWS:
```
Comment (Alan18081) [inline src/redux/news/news.reducer.js:14]: You don't need to handle this action type here, it is useless

File: src/redux/rootReduser.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import { combineReducers } from 'redux';
-import news from './news/news.reduser';
+import News from './news/news.reducer';
```
Comment (Alan18081) [inline src/redux/rootReduser.js:2]: Please, rename it to the newsReducer. Key in store should still be News

File: src/pages/news/news-item/index.js (hunk 2 above)
Comment (VolodymyrTrach) [reply src/pages/news/news-item/index.js:2]: done

File: .env.exaple (hunk 1 above)
Comment (VolodymyrTrach) [reply .env.exaple:2]: done

File: src/pages/news/news-page/news.js (hunk 3 above)
Comment (VolodymyrTrach) [reply src/pages/news/news-page/news.js:12]: done? deleted dependency array

File: src/redux/news/news.reducer.js (hunk 4 above)
Comment (VolodymyrTrach) [reply src/redux/news/news.reducer.js:14]: deleted

File: src/redux/rootReduser.js (hunk 5 above)
Comment (VolodymyrTrach) [reply src/redux/rootReduser.js:2]: done

File: src/redux/rootSaga.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import { all } from 'redux-saga/effects';
-import watchNewsLoad from './news/news.sagas';
+import getNews from './news/news.sagas';
 
 export default function* rootSaga() {
-  yield all([watchNewsLoad()]);
+  yield all([getNews()]);
```
Comment (Alan18081) [inline src/redux/rootSaga.js:5]: Rename this saga from `getNews` to `newsSaga` because it is a general saga related to `news`

Comment (VolodymyrTrach) [reply src/redux/rootSaga.js]: renamed

**Pull request merged**